src/widgets/permissionEntryRow.js

```diff
@@ -31,19 +31,19 @@ var FlatsealPermissionEntryRow = GObject.registerClass({
         'permission',
         'box',
         'button',
-        'image',
     ],
-}, class FlatsealpermissionEntryRow extends Handy.PreferencesRow {
+}, class FlatsealpermissionEntryRow extends Adw.PreferencesRow {
```

Comment:
```suggestion
}, class FlatsealPermissionEntryRow extends Adw.PreferencesRow {
```

Noticed while debugging the issue :sweat_smile:

Reply:
Whats affected by this? Its a preexisting issue and I fear breaking something by changing this

Reply:
Nothing should be affected by this, there are no references to this with the exact casing, and the class is simply just passed to GObject.registerClass